curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/649a76c556abbb027db091e8.md

```diff
@@ -0,0 +1,274 @@
+---
+id: 649a76c556abbb027db091e8
+title: Step 2
+challengeType: 0
+dashedName: step-2
+---
+
+# --description--
+
+Create a variable named `displayArray` and assign it to an empty array. This array will be used to display the dice values on the screen.
```

Comment:
```suggestion
Create a variable named `displayArray` and assign it to an empty array. This array will be used to display the dice values on the screen.

Because you will later be reassigning this array to reset the display, use `let` to declare the variable.
```